earthaccess/__init__.py

```diff
@@ -46,7 +47,9 @@
     "download",
     "auth_environ",
     # search.py
+    "DataGranule",
```

Comment:
Is this for testing purposes or we want users use these classes?

Reply:
Since we return a list of them to the user from the public function `search_data`, the user should have access to the type.

Reply:
To clarify, the user could import from the `search` module, but for consistency with the plural names being at the top level, along with the functions themselves, it makes sense to include this as well.